cg/meta/backup/pdc.py

```diff
@@ -13,16 +25,31 @@
 class PdcAPI:
     """Group PDC related commands"""
 
-    def __init__(self, binary_path: str = None, dry_run: bool = False):
+    def __init__(self, binary_path: str, dry_run: bool = False):
         self.process: Process = Process(binary=binary_path)
         self.dry_run: bool = dry_run
 
-    def archive_file_to_pdc(self, file_path: str, dry_run: bool = False) -> None:
-        """Archive a file by storing it on PDC"""
-        command: list = DSMCParameters.ARCHIVE_COMMAND.copy()
-        command.append(file_path)
-        if not dry_run:
-            self.run_dsmc_command(command=command)
+    @classmethod
+    def validate_is_dsmc_running(cls) -> bool:
```

Comment:
I still think that validate methods should not return booleans, but maybe that refactoring is for later

Reply:
Changed to logging instead 👍